nautobot/extras/models/change_logging.py

```diff
@@ -184,24 +184,23 @@ def get_action_class(self):
 
     def get_next_change(self, user=None):
         """Return next change for this changed object, optionally restricting by user view permission"""
-        related_changes = self.get_related_changes().restrict(user, "view")
-        if user is not None:
-            related_changes = related_changes.restrict(user, "view")
+        related_changes = self.get_related_changes(user=user)
         return related_changes.filter(time__gt=self.time).order_by("time").first()
 
     def get_prev_change(self, user=None):
         """Return previous change for this changed object, optionally restricting by user view permission"""
-        related_changes = self.get_related_changes()
-        if user is not None:
-            related_changes = related_changes.restrict(user, "view")
+        related_changes = self.get_related_changes(user=user)
```

Comment:
Wait a second, now I'm confused here. Why are they both making the same call? Was that a mistake?

Reply:
They're both getting related changes and then one sorts by `time__lt` and the other `time__gt`

Reply:
Oh lord have mercy "view new changes" UI threw me off. Disregard.